composer.json

```diff
@@ -43,7 +43,7 @@
         "doctrine/coding-standard": "^9.0.2 || ^11.0",
         "phpbench/phpbench": "^0.16.10 || ^1.0 || dev-master",
         "phpstan/phpstan": "~1.4.10 || 1.9.4",
-        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.5",
+        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.5.28@dev",
```

Comment:
PHPUnit uses `doctrine/instantiator` as well and would prevent this installation of v2. We can revert back to stable as soon as sebastianbergmann/phpunit#5126 has been released.